scroll-animations-1/Overview.bs

```diff
@@ -1054,6 +1352,87 @@ spec:selectors-4; type:dfn; text:selector
 
 	ISSUE: Define application to time-driven animations.
 
+### Examples
+
+	<div class=example>
+		In the following example, each direct child of the `.scroller` element
+		reveals itself as it enters the scrollport instead of when entirely crossing it.
```

Comment:
Clarify what this looks like from the user's perspective: it's opaque once fully visible.